### ISSUE TITLE
Where are the Scala communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/scala#566]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Scala supported by one or more large organizations?
* Does Scala have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Scala? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Scala? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Scala? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Scala?
* Are there popular newsletters for Scala?
* Is Scala taught at programming bootcamps? (If so, what are some examples?)
* Is Scala taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Scala?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by ricemery** [comment]
* There is a list of Scala conferences listed at - https://scala-lang.org/events/ . The big Scala conference is Scala Days which is happening on June 19th this year - https://na.scaladays.org/?_ga=2.227107806.2145206688.1528172378-421985141.1528172378 .
* Some influential companies that use Scala are listed on - https://en.wikipedia.org/wiki/Scala_(programming_language)#References .
* Scala was designed at -  École polytechnique fédérale de Lausanne (EPFL) . So, yes it is taught in Universities. Here is a list of some others - https://www.quora.com/What-universities-use-Scala-in-their-computer-science-curriculum .
 * Not sure on bootcamps. There are workshops offered during the Scala Days conferences - https://na.scaladays.org/workshops .
* Mailing lists and chat rooms can be found at - https://www.scala-lang.org/community/ .
* Scala news - https://www.scala-lang.org/community/#sources-of-scala-news .
* User groups - https://www.scala-lang.org/community/#user-groups
* Reddit community - https://www.reddit.com/r/scala/ .

**Comment by kytrinyx** [comment]
This is great, thanks!

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼
